addons/website_forum/models/forum_post.py

```diff
@@ -161,6 +162,90 @@ class Post(models.Model):
         'Can Use Full Editor',
         compute='_compute_post_karma_rights', compute_sudo=False)
 
+    def _accepted_answer(self):
+        for answer in self.child_ids:
+            if answer.is_correct:
+                return {
+                    "text": answer.plain_content,
+                    "upvoteCount": answer.vote_count,
+                    "url": answer.website_url,
+                    "datePublished": answer.create_date.strftime("%Y-%m-%dT%H:%M:%SZ"),
+                    "author_name": answer.create_uid.name,
+                    "author_url": f"/forum/user/{ answer.create_uid.id }",
+                }
+        return False
+
+    def _suggested_answers(self):
+        ret = []
+        for answer in self.child_ids:
+            if not answer.is_correct:
+                ret.append({
```

Comment:
- I would add a limit here like 5 ? To discuss with the ~~PO~~ Guru I suppose.
- I would add a comment to say that the post are ordered by vote_count so it's clear the suggested posts are the most voted.

Reply:
sure

Reply:
https://www.youtube.com/watch?v=v7sXTRrFWE4